projects/common/src/constants/application-constants.ts

```diff
@@ -1,5 +1,6 @@
 export const enum ApplicationFeature {
   PageTimeRange = 'ui.page-time-range',
   FeatureDefaultTimeRangeMap = 'ui.feature-default-time-range-map',
-  ExplorerOrderBySupport = 'ui.explorer-order-by-support'
+  ExplorerOrderBySupport = 'ui.explorer-order-by-support',
+  ApplicationFlowLayout = 'ui.application-flow-layout'
```

Comment:
please make sure that this flag is set to false in hypertrace feature flag resolver service. it is in source app

Reply:
oh yeah, updated

Reply:
As discussed, please remove this flag from here